src/main/java/org/kitteh/irc/client/library/Client.java

```diff
@@ -321,6 +331,83 @@
     }
 
     /**
+     * Proides commands.
+     */
+    interface Commands {
+        /**
+         * Provides a new AWAY command.
+         *
+         * @return new away command
+         */
+        @Nonnull
+        AwayCommand away();
+
+        /**
+         * Provides a new CAP REQ command.
+         *
+         * @return new capability request command
+         */
+        @Nonnull
+        CapabilityRequestCommand capabilityRequest();
+
+        /**
+         * Provides a new MODE command.
+         *
+         * @return new mode command
+         */
+        @Nonnull
+        ChannelModeCommand mode(@Nonnull Channel channel);
+
+        /**
+         * Provides a new KICK command.
+         *
+         * @return new kick command
+         */
+        @Nonnull
+        KickCommand kick(@Nonnull Channel channel);
+
+        /**
+         * Provides a new MONITOR command.
+         *
+         * @return new monitor command
+         */
+        @Nonnull
+        MonitorCommand monitor();
+
+        /**
+         * Provides a new OPER command.
+         *
+         * @return new oper command
+         */
+        @Nonnull
+        OperCommand oper();
```

Comment:
not many people are going to need this command, and it's a one-time only thing. Personally, I think this should not be there.